src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+    UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        enterUserName(scanner);
+        enterBet(scanner);
+        scanner.close();
+    }
+
+    private void enterUserName(Scanner scanner) {
+        while (true) {
+            System.out.println("Enter your name:");
+            String name = scanner.nextLine();
+            if (!name.isEmpty() && name.matches("[A-Za-z]+")) {
+                userDao.add(new User(name));
+                return;
+            }
+            System.out.println("Please enter correct data.");
+        }
+    }
+
+    private void enterBet(Scanner scanner) {
+        while (true) {
+            System.out.println("Enter value and risk for your bet:  (enter 'q' to exit)");
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
```

Comment:
let's not create Bet if the input was empty

Reply:
if the input was empty Bet will not be created
we will not get to the line where Bet is created